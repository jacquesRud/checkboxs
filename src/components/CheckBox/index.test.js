import React from "react";
import { render } from "@testing-library/react";
import Checkbox from "./";

describe("Checkbox", () => {
  test("False", () => {
    const { container } = render(<Checkbox enabled={false} />);
    expect(container).toMatchSnapshot();
  });
  test("True", () => {
    const { container } = render(<Checkbox enabled={true} />);
    expect(container).toMatchSnapshot();
  });
});
