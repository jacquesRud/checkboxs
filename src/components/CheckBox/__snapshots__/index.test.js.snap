// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox False 1`] = `
<div>
  <div
    style="cursor: pointer;"
  >
    [ 
     
     ]
  </div>
</div>
`;

exports[`Checkbox True 1`] = `
<div>
  <div
    style="cursor: pointer;"
  >
    [ 
    X
     ]
  </div>
</div>
`;
